kvbc/include/KVBCInterfaces.h

```diff
@@ -135,7 +135,8 @@ class ICommandsHandler : public bftEngine::IRequestsHandler {
                       const char* request,
                       uint32_t maxReplySize,
                       char* outReply,
-                      uint32_t& outActualReplySize) = 0;
+                      uint32_t& outActualReplySize,
+                      concordUtils::SpanWrapper& span) = 0;
```

Comment:
This is going to be a breaking change for the product. So right after merging this into master, please also bump concord-bft in the product and make the necessary changes.

Reply:
Yes, I have a commit for the product.